bin/zero-parachain/runtime/src/constants.rs

```diff
@@ -0,0 +1,93 @@
+pub mod time {
+	use primitives::{BlockNumber, Moment};
+
+	pub const SECS_PER_BLOCK: Moment = 12;
+	pub const MILLISECS_PER_BLOCK: Moment = SECS_PER_BLOCK * 1000;
+
+	// These time units are defined in number of blocks.
+	pub const MINUTES: BlockNumber = 60 / (SECS_PER_BLOCK as BlockNumber);
+	pub const HOURS: BlockNumber = MINUTES * 60;
+	pub const DAYS: BlockNumber = HOURS * 24;
+
+	pub const SLOT_DURATION: Moment = MILLISECS_PER_BLOCK;
+}
+
+pub mod fee {
+	use frame_support::weights::{
+		constants::{ExtrinsicBaseWeight, WEIGHT_PER_SECOND},
+		WeightToFeePolynomial,
+		WeightToFeeCoefficients,
+		WeightToFeeCoefficient
+	};
+	use smallvec::smallvec;
+	use sp_runtime::Perbill;
+	use primitives::{
+		currency::{TokenSymbol, DOT, ZERO, GAME, PLAY},
+		Balance,
+		cent
+	};
+
+	pub fn base_tx_in_token(token: TokenSymbol) -> Balance {
+		match token {
+			TokenSymbol::DOT => (cent(DOT) * 30 / 10000),
+			// ZERO:DOT = 6:1
+			TokenSymbol::ZERO => (cent(ZERO) * 18 / 1000),
+			// GAME:DOT = 6:1
+			TokenSymbol::GAME => (cent(GAME) * 18 / 1000),
+			// PLAY:DOT = 6:1
+			TokenSymbol::PLAY => (cent(PLAY) * 18 / 1000),
```

Comment:
@2075 could you review this? It's the initial price ratio.

Reply:
lgtm, how would price discovery via acala dex work? alternative could be through fx provider via ocw

Reply:
Price discovery is separate piece of logic.
Here proportions determined to know amount of fee to pay per cross-chain transactions for each known token.
In case we would need to change those, should do runtime upgrade - same approach as for other parachains.